website/docs/r/nat_gateway.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_nat_gateway"
+sidebar_current: "docs-azurerm-resource-nat-gateway"
+description: |-
+  Manage Azure NatGateway instance.
+---
+
+# azurerm_nat_gateway
+
+Manage Azure NatGateway instance.
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Nat Gateway. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group. Changing this forces a new resource to be created.
+
+* `location` - (Optional) Specifies the supported Azure location where the resource should exist. Changing this forces a new resource to be created.
+
+* `idle_timeout_in_minutes` - (Optional) The idle timeout of the Nat Gateway. Defaults to `4`.
+
+* `public_ip_address_ids` - (Optional) A list of existing `azurerm_public_ip` resource IDs.
+
+* `public_ip_prefix_ids` - (Optional) A list of existing `azurerm_public_ip_prefix` resource IDs.
+
+* `resource_guid` - (Optional) The resource GUID property of the Nat Gateway.
+
+* `sku_name` - (Optional) The nat gateway SKU, supported value, `Standard`. Defaults to `Standard`.
+
+* `zones` - (Optional) A list of availability zones where the Nat Gateway should be provisioned. Supported values are `1`, `2`, and `3`. For more information on `zones` please refer to the [product documentation](https://docs.microsoft.com/en-us/azure/availability-zones/az-overview). Changing this forces a new resource to be created.
+
+* `tags` - (Optional) A mapping of tags to assign to the resource. Changing this forces a new resource to be created.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `subnet_ids` - A list of `azurerm_subnet` resource IDs.

```

Comment:
Could we add a little more details here what the property does, not just what it is?

Reply:
we are also missing the import docs

Reply:
I added more details about this